### DISCUSSION TITLE
Select multiple items and apply properties to all based on single torrent's properties

### DISCUSSION THREAD

**Discussion by iatacs19** [openscopeproject/TrguiNG#57] [Ideas]
In the original Transmission GUI, you are able to select multiple torrents then right-click on a single torrent and select Properties, then when you click OK, it will apply that select torrent's properties to all the highlighted torrents. Would this be something that is feasible?

**Comment by qu1ck** [comment]
It is feasible but I didn't implement it because it can easily break your torrents if you accidentally overwrite the private tracker urls for example. And I didn't find a good use case that would provide a benefit/counterweight to that risk. 

What is your use case for wanting to batch update torrent properties? Maybe it is something that can be achieved by changing session level defaults instead?

**Comment by iatacs19** [comment]
Thank you for the reply.
I usually download from several trackers and once I reach a certain ratio, I manually change the max up/down/ratio values for the completed torrents to an alternate value. It's handy when I am changing many torrents at once.

**Comment by tml** [comment]
Another use case is if a private tracker changes its endpoint, it's nice to be able to edit/update all the torrents pointing to the old endpoint in one go.

**Comment by Antresa** [comment]
I would also like this feature. I need to change the announce URL on hundreds of torrents and not being able to make a batch update is a real pain.

**Comment by qu1ck** [comment]
This is now possible in [v0.10.0](https://github.com/openscopeproject/TrguiNG/releases/tag/v0.10.0).

**Reply by Antresa** [reply]
That was quick. Thank you very much!

**Reply by iatacs19** [reply]
Thank you as well!
